src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,78 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Nested;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CalculatorTest {
+
+    private final Calculator calculator = new Calculator();
+
+    @Nested
+    @DisplayName("덧셈 테스트")
+    class addTest {
+
+        @Test
+        @DisplayName("정수의 합")
+        void add_Test() {
+            final int a = 3;
+            final int b = 5;
+            final int actual = a + b;
+
+            assertEquals(actual, calculator.add(a, b));
+        }
+    }
+
+    @Nested
+    @DisplayName("뺄셈 데스트")
+    class subtractTest {
+
+        @Test
+        @DisplayName("정수의 차")
+        void subtract_Test() {
+            final int a = 5;
+            final int b = 3;
+            final int actual = a - b;
+
+            assertEquals(actual, calculator.subtract(a, b));
+        }
+    }
+
+    @Nested
+    @DisplayName("곱셈 테스트")
+    class multiplyTest {
+
+        @Test
+        @DisplayName("정수의 곱셈")

```

Comment:
각 테스트 메서드가 어떤 상황을 테스트하는 지 구체적으로 설명하면 좋을 것 같아요!

```suggestion
        @DisplayName("각 정수의 곱한 값을 반환한다.")
```

또한, 곱셈의 예외 상황에 대해서도 테스트하면 더 좋을 것 같습니다!

Reply:
아래와 같이 곱셈에 대한 오버 플로우 예외 상황을 적용시킬 수 있을 거 같습니다 
또한 테스트메서드를 직관적으로 설명할 수 있게 노력해보도록 하겠습니다 감사합니다 !

```java
public int multiply(int num1, int num2) {
    long result = (long) num1 * num2;
    if (result > Integer.MAX_VALUE) {
        throw new ArithmeticException("오버플로우가 발생했습니다.");
    }
    return (int) result;
}

@Test
@DisplayName("곱셈 예외 테스트")
void multiply() {
    int a = Integer.MAX_VALUE;
    int b = 2;
    assertThrows(ArithmeticException.class, () -> {
        calculator.multiply(a, b);
    });
}